theme/table.css

```diff
@@ -3,6 +3,10 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
+ :root {
```

Comment:
Do we need a variable for this, if it's used in one place?

Reply:
I can see that you actually used it in https://github.com/ckeditor/ckeditor5-theme-lark/pull/271/files. Which IMO is wrong because you named it `th-background` and yet you use it for focused cell background. The variable has a meaning and you should use it according to its meaning.

Anyway, I don't think we need a variable for that. The ticket was about switching to an hsla color. Refactor should be justified. So the question is – why did you refactor it? Do we need to do that? If no, let's focus on the color itself.

Reply:
> you named it `th-background` and yet you use it for focused cell background

Actually, it's the other way around – you called it "focused th background" and you use it for normal ths :P

Reply:
You're right, sorry about that 👍